scss/_patterns_links.scss

```diff
@@ -11,6 +11,40 @@
     &--no-underline {
       border: 0;
     }
+
+    &--soft {
+      color: $color-dark;
+
+      &:hover {
+        color: $color-accent;
+      }
```

Comment:
You are missing the `is-selected` styling. The design shows that is selected should be 400 font weight.
https://raw.githubusercontent.com/ubuntudesign/vanilla-design/master/Links/Link%20styles%20Vanilla.png